src/_Plugin_Helper.cpp

```diff
@@ -128,3 +128,10 @@ bool pluginOptionalTaskIndexArgumentMatch(taskIndex_t taskIndex, const String& s
   }
   return found_taskIndex == taskIndex;
 }
+
+int getValueCountForTask(taskIndex_t   taskIndex) {
+  struct EventStruct TempEvent;
+  String dummy;
+  PluginCall(PLUGIN_GET_DEVICEVALUECOUNT, &TempEvent, dummy);
+  return TempEvent.Par1;
+}
```

Comment:
@TD-er maybe return something default if not handled?

Reply:
never mind, it is already handled ;)

Reply:
Still glad someone is looking at my code :)

Reply:
@TD-er my amount of time to do something interesting or just to play with any code is almost zero now, but ... my eyes are still here :)